docusaurus/docs/Flutter/05-advanced/08-custom-data.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Custom Data
+slug: /custom-data
+sidebar_position: 8
+description: Learn how to add and read custom data in the Stream Video Flutter SDK.
+---
+
+Custom data is additional information that can be added to the default data of Stream. It is a dictionary of key-value pairs that can be attached to users, events, and pretty much almost every domain model in the Stream SDK.
+
+On Flutter, custom data is represented by the `Map<String, Object>`. This means that the key must be a string and the value can be any object.
```

Comment:
Maybe let's say "in the SDK" or "in calls for stream video" rather than "on flutter"